packages/legend-studio/src/stores/editor-state/element-editor-state/mapping/MappingEditorState.ts

```diff
@@ -291,7 +292,7 @@ export const createEnumerationMapping = (
 const getEmbeddedSetImplmentations = (
   mapping: Mapping,
 ): InstanceSetImplementation[] =>
-  mapping.allClassMappings
+  mapping.allOwnClassMappings
```

Comment:
Could you document this real quick, feel like this behavior for embedded is quite specialized